examples/wallet/app/actions/router.js

```diff
@@ -3,10 +3,17 @@ import { push } from 'connected-react-router';
 import type { Dispatch, GetState } from '../reducers/types';
 import type { Address } from '../models';
 import routes from '../constants/routes';
+import { setAccountFromPrivateKey } from './account';
+
+import { readAccountKeysFromLocalStorage } from '../utils/storage';
 
 // eslint-disable-next-line import/prefer-default-export
 export const redirectToFirstAppPage = () => {
   return (dispatch: Dispatch, getState: GetState) => {
+    const accountKeys = readAccountKeysFromLocalStorage();
+    if (accountKeys !== undefined) {
```

Comment:
`if(accountKeys)` would be enough and IMO more idiomatic

Reply:
corrected